api/src/convert/block_metadata.rs

```diff
@@ -24,13 +29,27 @@ impl TryFrom<&blockchain::BlockMetadataContents> for BlockMetadataContents {
     fn try_from(src: &blockchain::BlockMetadataContents) -> Result<Self, Self::Error> {
         let block_id = src.get_block_id().try_into()?;
         let quorum_set = src.get_quorum_set().try_into()?;
-        let report = src.get_verification_report().try_into()?;
+        let attestation_evidence = match (src.has_dcap_evidence(), src.has_verification_report()) {
```

Comment:
This shouldn't be necessary - I suggest looking at the generated code (search for `blockchain.rs` under `target`).
You'll find an enum called `BlockMetadataContents_oneof_attestation_evidence`, and you will be able to match against `src.attestation_evidence` which is an `Option<BlockMetadataContents_oneof_attestation_evidence>`

Reply:
Logic has been updated to look at the `attestation_evidence` field of the `blockchain::BlockMetadataContents`.

I didn't realize they were public fields, thanks!